xbmc/interfaces/legacy/Player.h

```diff
@@ -519,6 +519,24 @@ namespace XBMCAddon
       void seekTime(double seekTime);
 #endif
 
+#ifdef DOXYGEN_SHOULD_USE_THIS
+      ///
+      /// \ingroup python_Player
+      /// @brief \python_func{ addSubtitle(subtitleFile, name, language, activate) }
+      ///-----------------------------------------------------------------------
+      /// Add subtitle file with name and language and optionally activate it.
+      ///
+      /// @param subtitleFile        File to use as source ofsubtitles
+      /// @param name                Name of the subtitle to display
+      /// @param language            Language of the subtitlte to use
+      /// @param activate            Whether or not to activate the subtitle
+      ///
+      addSubtitle(...);
```

Comment:
Misses python api revision information: `@python_v19 New function added.`.
Copy&paste:

```
///
/// \ingroup python_Player
/// @brief \python_func{ addSubtitle(subtitleFile, name, language, activate) }
///-----------------------------------------------------------------------
/// Add subtitle file with name and language and optionally activate it.
///
/// @param subtitleFile        File to use as source ofsubtitles
/// @param name                Name of the subtitle to display
/// @param language            Language of the subtitlte to use
/// @param activate            Whether or not to activate the subtitle
///-----------------------------------------------------------------------
/// @python_v19 New function added.
///
```

Thanks a lot for the feature

Reply:
Thanks i'll fix it.

PS: Github has the "Insert a suggestion" (button left of the formatting buttons or Ctrl + G) feature which allows you to propose a change to the reviewed code. The author of the PR can then directly take over the suggestion.